frontend/src/components/elements/ExceptionElement/__snapshots__/ExceptionElement.test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`ExceptionElement Element renders without crashing 1`] = `"<div class=\\"alert exception stException alert-danger\\" style=\\"width:0\\"><div class=\\"message\\"><span class=\\"type\\">RuntimeError</span>: This is an exception of type RuntimeError</div><div class=\\"stack-trace-title\\">Traceback:</div><pre class=\\"stack-trace\\"><code><div class=\\"stack-trace-row\\">step 1</div><div class=\\"stack-trace-row\\">step 2</div><div class=\\"stack-trace-row\\">step 3</div></code></pre></div>"`;
+exports[`ExceptionElement Element renders without crashing 1`] = `"<div class=\\"alert exception stException alert-danger\\" style=\\"width: 0px;\\"><div class=\\"message\\"><span class=\\"type\\">RuntimeError</span>: This is an exception of type RuntimeError</div><div class=\\"stack-trace-title\\">Traceback:</div><pre class=\\"stack-trace\\"><code><div class=\\"stack-trace-row\\">step 1</div><div class=\\"stack-trace-row\\">step 2</div><div class=\\"stack-trace-row\\">step 3</div></code></pre></div>"`;
```

Comment:
This is a weird change. Where is it from?

Reply:
when i made the component 'mount' instead of 'shallow', it caused a change in the expected snapshot. something to do with enzyme probably